src/main/java/seedu/trackascholar/model/applicant/Applicant.java

```diff
@@ -63,22 +63,42 @@ public Name getName() {
         return name;
     }
 
+    public String getFullName() {
+        return name.getFullName();
+    }
+
     public Phone getPhone() {
         return phone;
     }
 
+    public String getPhoneNumber() {
+        return phone.getNumber();
```

Comment:
Same thing here, except instead of disambiguating, it's more for standardizing getter names.

Reply:
Noted, changes made